packages/lib/src/vis/line/models.ts

```diff
@@ -1,20 +1,15 @@
-import capURL from './sprites/cap.png';
-import circleURL from './sprites/circle.png';
-import crossURL from './sprites/cross.png';
-import squareURL from './sprites/square.png';
```

Comment:
In `jest.config.js` of @h5web/app, there's a `transform` config to handle those. We should be able to remove it.